src/meta/client/MetaClient.h

```diff
@@ -260,6 +313,10 @@ class MetaClient {
     MetaChangedListener*  listener_{nullptr};
     bool                  sendHeartBeat_ = false;
     std::atomic_bool      ready_{false};
+    MetaConfigMap         metaConfigMap_;
+    folly::RWSpinLock     configCacheLock_;
+    thread::GenericWorker loadCfgThread_;
```

Comment:
Could we reuse current background thread?

Reply:
Of course, but I think maybe we need to different load interval?